alvr/server/cpp/platform/linux/utils.hpp

```diff
@@ -0,0 +1,64 @@
+#pragma once
```

Comment:
What exactly is this used for? I also think this should go into the shared directory

Reply:
I needed an actually memory stable optional because of some old recycled alvr code throwing pointers to stuff around, I'll check if I still need it. Also because it's not externally maintained code it should not go into shared